guides/acronyms.md

```diff
@@ -0,0 +1,57 @@
+# DVELP Acronyms
+
+The below is a list of acronyms that are commonly used in the work we do, along
+with their definitions. If you come across an acronym you don't know this is a
+good place to reference.
+
+| Acronym | Definition |
+| --- | --- |
+| AGM | Annual General Meeting |
+| ASR | Automated Speech Recognition |
+| Autopilot | Twilio Product? |
+| BAU | Business as usual |
+| BCM | Business Continuity Management |
+| canned responses | pre-determined and saved response, ready to be called programatically. E.g. "Please hold...|
+| CLI | Command line interface |
+| DBS | Disclosure and Barring Service (Police background check) |
+| DDI | Direct Dial-In, dedicated number (used in the background, not linked to MPOA) |
+| DDOS | Distributed Denial of Service Attack |
+| DOS | Denial of service attack |
+| DPIA | Data Protection Impact Assessment |
+| DPIA | Data Protection Impact Assessments |
+| DPO | Data Protection Officer |
+| DTMF | Pressing button on dialphone to direct call during call (Dual-tone multi-frequency signaling) |
+| E164 | aka long number e.g. +447500172268 |
+| FOIA | Freedom of Information Act |
+| GDPR | General Data Protection Regulation |
+| GTM | Go to market |
+| ICO | Information Commissioner's Office |
+| IEC | International Electrotechnical Commission (Standardisation body linked to ISO) |
+| ISMS | Information Security Management System |
+| ISO | International Organization for Standardization |
+| ISV  | Independent Software Vendor |
+| IVR | Interactive Voice Response |
+| MPOA | Publicly displayed number |
+| MQL | Marketing Qualified Lead |
+| MSA | Master Service Agreement |
+| Multi-modal | e.g. starting on phone, using SMS/email/app to comlete a process |
+| NDA | Non-disclosure Agreement |
+| NLR | Natural Language Recognition |
+| OWASP | Open Web Application Security Project www.owasp.org |
+| PBX | Private Branch Exchange, which is a private telephone network used within a company or organization to switch calls internally |
+| PCI DSS | Payment Card Industry Data Security Standard  |
+| PDP | Personal Development Programme |
+| Predictive dialer | A predictive dialer dials a list of telephone numbers and connects answered dials to people making calls, often referred to as agents. |
```

Comment:
Title case. Also, I think this is no longer the correct definition, given what we learned?

Reply:
Good point. Do you agree with the new definition?